docs/tutorials/creating-a-database-adapter.md

```diff
@@ -36,7 +36,7 @@ export default function MyAdapter(client, options = {}) {
     async linkAccount(account) {
       return
     },
-    async unlinkAccount({ provider, id }) {
+    async unlinkAccount({ providerAccountId, provider}) {
```

Comment:
```suggestion
    async unlinkAccount({ providerAccountId, provider}) {
```

This method currently isn't even invoked in the core, but we will likely match it with `getUserByAccount`, good catch!